in_toto/models/layout.py

```diff
@@ -120,6 +120,27 @@ def read(data):
 
     return Layout(**data)
 
+  def display(self):
+    """
+    <Purpose>
+      Represents the Layout class by returning its fields in a readable string
+    <Returns>
+      Layout fields: type, expiration, readme, keyids, steps, inspections
+    """
+    types = "object: {}".format(self._type)
+    expiration = "expires: {}".format(self.expires)
+    if not self.readme:
+      readme = "readme: None"
+    else:
+      readme = self.readme
+    keyid = "keyids: {}".format("\n\t  ".join(self.keys))
+
+    steps = "step data:\n {}".format(self.steps)
+    inspections = "inspect data:\n {}".format(self.inspect)
+
+    return "  {}\n  {}\n  {}\n  {}\n  {}\n  {}".format(types, expiration, readme, 
```

Comment:
IMHO it would be more readable and easier to change if you defined a format string that already looks somewhat like what you will return, e.g.:

```python
return (
  "object: {}\n"
  "expiration: {}\n"
  "{}\n"
  "keyids: {}\n"
  "..."
  .format(self._type, self.exipration, self.readme, keyids .... 
)
```
What do you think?

Reply:
Yup, this seems spot on. I've updated the code to look more like this.